agent/engine/task_manager.go

```diff
@@ -1361,7 +1361,7 @@ func (mtask *managedTask) containerNextState(container *apicontainer.Container)
 	}
 	if blocked, err := dependencygraph.DependenciesAreResolved(container, mtask.Containers,
 		mtask.Task.GetExecutionCredentialsID(), mtask.credentialsManager, mtask.GetResources(), mtask.cfg); err != nil {
-		logger.Debug("Can't apply state to container yet due to unresolved dependencies", logger.Fields{
+		logger.Info("Can't apply state to container yet due to unresolved dependencies", logger.Fields{
```

Comment:
Good catch, should we also evaluate whether the other logs within `containerNextState()` should be INFO or not?

Reply:
Happy for this to be a follow up (non-blocking)

Reply:
Yeah we should. This particular log would have been useful for a recent issue so I am doing this one first.